bin/inference/pycbc_inference_plot_movie

```diff
@@ -134,7 +138,7 @@ for frame in range(nframes):
         z = zvals[:,frame]
     else:
         z = None
-    sample_num = str(frame * thinint + 1)
+    sample_num = str(frame * thinint + opts.start_sample)
```

Comment:
No, I think it's better to keep the number of samples be the same thing as the iteration number. Otherwise it can get confusing.